scalableminds/webknossos
# Fix Dataset Configurations

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open Dataset in View Mode and change segmentation settings (e.g. opacity)
- open Volume tracing without fallback on the same dataset and change volume layer settings (e.g. opacity)
- reopen Dataset in View Mode => segmentation settings should persist
- reopen tracing => volume layer settings should persist

### Issues:
- fixes #4650 

------
